oshi-json/src/main/java/oshi/json/hardware/HWDiskStore.java

```diff
@@ -81,22 +83,33 @@
      *            milliseconds since the epoch
      */
     public HWDiskStore(String name, String model, String serial, long size, long reads, long readBytes, long writes,
-            long writeBytes, long transferTime, HWPartition[] partitions, long timeStamp) {
+            long writeBytes, long currentQueueLength, long transferTime, HWPartition[] partitions, long timeStamp) {
```

Comment:
Replace this constructor with one that takes an `oshi.hardware.HWDiskStore` argument and sets values based on the argument's getters.

Reply:
Hm, not sure what you mean? Do you want to have a copy constructor?

Reply:
See `HardwareAbstractionLayerImpl.java` line 136 where you added another param to the constructor.  Consistent with the goal to remove multi-arg constructors you still need a way for that Impl class to wrap the `oshi-core` object inside the `oshi-json` object.  So you will create a new `oshi.json.hardware.HWDiskStore` using an `oshi.hardware.HWDiskStore` argument.  Then for each of the elements in the `oshi-core` version, use its getter inside the setter for the `oshi-json` version.

Reply:
pushed changes